pkgs/by-name/so/soplex/package.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, cmake
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Have you cogitated `finalAttrs` design pattern?

Reply:
TIL -- thanks!